docs/dev-environments/ethereum/besu/docker-compose-truffle.yaml

```diff
@@ -24,7 +24,7 @@ services:
         - ./my_project:/my_project
         - $TCF_HOME/sdk/avalon_sdk/ethereum/contracts/proxy-model:/contracts
         - $TCF_HOME/sdk/avalon_sdk/ethereum/truffle_artifacts:/truffle_artifacts
```

Comment:
Can these files be made part of container image by COPY in Dockerfile rather than volume mount.  Though volume mount is helpful, it requires user to have a copy of Avalon code base locally. Going forward we also want to support K8s for Avalon and at that point it's not feasible to assume that Avalon code base will be downloaded in every K8s node.